transport.go

```diff
@@ -36,6 +38,10 @@ func (t defaultTransport) get(ctx context.Context, url string) ([]byte, error) {
 	}
 	defer closeSilently(resp.Body)
 
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("http code %d is not OK", resp.StatusCode)
```

Comment:
This way we cannot distinguish what was exact OSRM error code (since it is passed in response body according to http://project-osrm.org/docs/v5.5.1/api/#responses ), right?

Reply:
It was discussed. I'll refactor this part.